Sources/Core/Sources/Render/Shader/ChunkShaders.metal

```diff
@@ -89,3 +89,38 @@ fragment float4 chunkFragmentShader(RasterizerData in [[stage_in]],
 
   return color;
 }
+
+
+constant float2 quadVertices[] = {
+     float2(-1.0,  1.0),
+     float2(-1.0, -1.0),
+     float2( 1.0, -1.0),
+     float2(-1.0,  1.0),
+     float2( 1.0,  1.0),
+     float2( 1.0, -1.0)
+ };
+
+ struct QuadVertex {
+     float4 position [[position]];
+     float2 uv;
+ };
+
+
+ fragment float4 screenFragmentFunction(QuadVertex v [[stage_in]],
+                                  texture2d <float> outputImage [[ texture(0) ]]) {
+
+     constexpr sampler smplr(coord::normalized);
```

Comment:
Use 2 space indentation not 4 space indentation here

Reply:
My bad, switching between 2-4 indentation all the time 😬